src/api/post/asyncGetPosts.js

```diff
@@ -5,16 +5,18 @@ const asyncGetPosts = async (
   cursorId = POST_LISTS.DEFAULT_CURSOR_ID,
   {
     keywordId,
+    order = POST_LISTS.DEFAULT_ORDER,
```

Comment:
```suggestion
    order = POST_LISTS.ORDER_EN.NEWEST,
```
저는 개인적으로 상수명만으로도 어떤 데이터를 가지고 있는지 파악되어야 한다고 생각하는데요. 
현재 선언해주신 상수명은 디폴트값인 것은 파악되지만, 해당 디폴트값이 빈문자열인지 아니면 특정 값을 가지고 있는지 유추가 되지 않아 상수 파일을 확인해야 하는 번거로움이 있는 것 같아요..!
현재 디폴트값으로 선언되어있는 "NEWEST"는 이미 `POST_LISTS.ORDER_EN.NEWEST` 로 선언되어 있기 때문에, 해당 상수로 바로 선언해주는게 좋을 것 같은데, 어떻게 생각하시나요?!

Reply:
좋은 고민 남겨주셔서 감사해요. 데이터를 가늠할 수 있는 상수명은 좋다고 생각합니다. 다만 데이터의 default 기준은 바뀔 수 있는데, 이때 현재 구조를 유지하면 '상수에서만 변경해주면 된다'는 장점이 있다고 생각합니다!

이러한 이유로 아래 처럼 저희 컨벤션도 상수명에서 데이터를 구체적으로 특정하지 않도록 한다고 이해하고 있었어요. 구체적으로 명시하면 해당 상수가 쓰인 모든 파일에서 누락 없이 수정해야 한다는 번거로움이 있습니다. 그래서 저희가 기본적인 정책을 이해하고 있다면 현재 처럼 명시해도 무리가 없다고 생각하는데, @SsongQ-92 규경님과도 함께 검토해보면 좋겠습니다 :)
```js
// asyncGetPosts 함수 parameter
{
    keywordId,
    order = POST_LISTS.DEFAULT_ORDER,
    includedKeyword = POST_LISTS.DEFAULT_INCLUDED_KEYWORD,
    excludedKeyword = POST_LISTS.DEFAULT_EXCLUDED_KEYWORD,
    isAd = POST_LISTS.DEFAULT_IS_AD,
    limit = POST_LISTS.DEFAULT_LIMIT,
  }
```

Reply:
진태님 의견에도 동의합니다. 
다만, 제가 혼란스러웠던 것은 답변에 작성해주신 코드에서 선언된 6개 변수 중 5개 변수의 디폴트 값으로 상수값이 설정되어 있는데요. 
디폴트값으로 특정한 값이 선언되어 있는 경우엔 진태님이 말해주신대로 값이 변경된다면 상수 파일만 수정하면 된다는 장점으로 상수화시키는 것은 좋다고 생각합니다. 
  ```jsx
  DEFAULT_ORDER: "NEWEST"
  ```
그러나 특정한 값이 아닌 빈문자열과 빈배열은 오히려 상수화하지 않는 것이 더 직관적이지 않을까 생각해요. 상수보단 형태 그대로 표시하는 것이 가독성 측면에서 좋다고 판단되었어요. 
해당 값들에 대해 상수화를 유지한다면, `DEFAULT` 보단 `EMPTY`라는 단어가 들어가는 것이 의미를 바로 이해할 수 있을 것 같기도 합니다.
  ```jsx
  DEFAULT_INCLUDED_KEYWORD: [],
  DEFAULT_IS_AD: "",
  ```
해당 부분에 대해서 규경님과도 논의하면 좋을 것 같습니다!

Reply:
저는 해당 부분의 기능적인 부분에 맞춰서 생각을 해봤는데요, 위의 상수화된 변수명은 파라미터의 기본값을 명시하기 위해 만들어진 변수이기 때문에, 아래와 같이 하는 것이 좋을 것 같다고 생각합니다. 이전에도 비슷한 얘기가 나왔던 것 같은데요, `DEFAULT_ORDER`라고 하고, 협업의 입장에서 생각했을 때, `constants` 파일에서 해당 부분의 기본값을 보거나 바꿔줄 수 있기 때문에 충분히 변수명이 구체적이라고 생각합니다.

하지만, 앞으로 바뀌지 않을 고정된 빈 문자열의 경우, 저희가 상수화하지 않기로 약속했던 것 같아요. 

```js
{ 
  keywordId, 
  order = POST_LISTS.DEFAULT_ORDER, 
  includedKeyword = POST_LISTS.DEFAULT_INCLUDED_KEYWORD, 
  excludedKeyword = POST_LISTS.DEFAULT_EXCLUDED_KEYWORD, 
  isAd = POST_LISTS.DEFAULT_IS_AD, 
  limit = POST_LISTS.DEFAULT_LIMIT, 
}
```

Reply:
구두로 논의한대로, 해당 방향으로 수정했습니다. 감사합니다.